src/fhir/utilities.ts

```diff
@@ -284,4 +284,224 @@ export class FhirUtilities {
       });
     });
   }
+
+  static async populateDB() {
+    const db = Globals.database;
+
+
+    // define schemas
+    const medicationCollection = await db.collection('medication-requirements'
+    // , {
+    //   'name': { 'type': 'string' },
+    //   'codeSystem': { 'type': 'string' },
+    //   'code': { 'type': 'string' },
+    //   'requirements': {
+    //     'type': 'array',
+    //     'items': {
+    //       'type': 'object',
+    //       'properties': {
+    //         'name': { 'type': 'string' },
+    //         'description': { 'type': 'string' },
+    //         'questionnaire': { 'type': 'object' },
+    //         'stakeholderType': { 'type': 'string' },
+    //         'createNewCase': { 'type': 'boolean' },
+    //         'resourceId': { 'type': 'string' }
+    //       }
+    //     }
+    //   }
+    // }, (err: any, collection: any) => {
+    //   if (err) console.log(err);
+    // }
+    );
+
+
+    await medicationCollection.createIndex({ name: 1 }, { unique: true });
+
+    const metRequirementsCollection = await db.collection('met-requirements'
+    // , {
+    //   'completed': { 'type': 'boolean' },
+    //   'completedQuestionnaire': { 'type': 'object' },
+    //   'requirementName': { 'type': 'string' },
+    //   'requirementDescription': {'type': 'string'}
+    //   'drugName': { 'type': 'string' },
+    //   'stakeholderId': { 'type': 'string' },
+    //   'case_numbers': { 'type': 'array', 'items': { 'type': 'string' } }
+    // }, (err: any, collection: any) => {
+    //   if (err) console.log(err);
+    // }
+    );
+
+
+    metRequirementsCollection.createIndex({ drugName: 1, requirementName: 1, stakeholderId: 1 }, { unique: true });
+
+
+    const remsCaseCollection = await db.collection('rems-case'
+    // , {
```

Comment:
Are these comments to show the structure?
If so maybe a comment stating that would be good. Otherwise it should be removed.